src/assets/toolkit/scripts/lib/component/fact-slider.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+export class FactSlider {
+  constructor (element, {
+    facts = element.querySelectorAll('.FunFact'),
+    factNavNext = element.querySelector('.js-FunFact-next'),
+    factNavPrev = element.querySelector('.js-FunFact-prev'),
+    factNavCurrent = element.querySelector('.js-FunFact-current'),
+    factNavTotal = element.querySelector('.js-FunFact-total'),
+    className = 'is-visible'
+  } = {}) {
+
+    facts = Array.from(facts);
+    let numFacts = facts.length;
+    let counter = 0;
```

Comment:
I'm not following this one. I did try this, but then things break...This was one of the things that was confusing me...not sure where/how to define variables like this.